src/interfaces/ISubRegistry.sol

```diff
@@ -22,6 +22,14 @@ interface ISubRegistry {
             address[] memory
     );
 
+    function getAssetInformation(address asset) external
```

Comment:
This is identical to the previous one?

Reply:
going to blame this on merge issues I'll remove and push it